content/en/tracing/services/service_page.md

```diff
@@ -137,6 +137,30 @@ Choose what to display in your resources list:
 
 {{< img src="tracing/visualization/service/resource_columns.png" alt="Resource columns" style="width:40%;">}}
 
+{{< site-region region="ap1" >}}
+## Dependencies
```

Comment:
Note: We have a link to this section from **Overview**. We should wrap that in the site-region shortcode too so users outside of AP1 don't get confused:

Reply:
Done !